packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -194,6 +206,18 @@ export class InMemoryAnchorService implements AnchorService, AnchorValidator {
     })
   }
 
+  _startProcessingCandidate(candidate: Candidate, message?: string): void {
```

Comment:
this should probably be added to `_process` also to make the full anchor process behave more like the real CAS